mollie-payments-for-woocommerce/includes/mollie/wc/payment/RefundLineItemsBuilder.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * Class RefundLineItemsBuilder
+ */
+class RefundLineItemsBuilder
+{
+    /**
+     * @var Mollie_WC_Helper_Data
+     */
+    private $dataHelper;
+
+    /**
+     * RefundLineItemsBuilder constructor.
+     * @param Mollie_WC_Helper_Data $dataHelper
+     */
+    public function __construct(Mollie_WC_Helper_Data $dataHelper)
+    {
+        $this->dataHelper = $dataHelper;
+    }
+
+    /**
```

Comment:
Had to read and understand the method in order to know what it does. Docs would be nice.

Reply:
I improved some docblock but not sure about inner exceptions, when them are too generic and them go up in the call chain could start to be difficult to maintain them. I would leave as it is for now.